src/app/public/modules/colorpicker/colorpicker.component.scss

```diff
@@ -245,3 +245,34 @@
     }
   }
 }
+
+.checkered-background {
```

Comment:
Looks good! Mind prefixing the class name with `sky-colorpicker-`?

Reply:
renamed :)